db/migrations/004.do.active-deals-timestamps.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE active_deals 
+    ADD COLUMN IF NOT EXISTS submitted_at TIMESTAMP,
+    ADD COLUMN IF NOT EXISTS created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP;
```

Comment:
If would consider replacing `created_at` with `term_start_epoch` If that's possible? How often will`term_start_epoch` be set to some date in the past if we're listening for real-time events?

Reply:
I'd like the database to store the raw data we are getting from the Filecoin on-chain state, and calculate the timestamps from that. The algorithm for calculating the timestamps can change in the future as we gain more understanding of the deal lifecycle and actor events. By keeping the raw data in our table, it's easy to change the way how we interpret it.

To find the time when a deal was created, you should look at `activated_at_epoch` and convert it to a timestamp.

Reply:
Thank you for this info! I've dropped `created_at` column 🙏🏻